protocol/testing/e2e/gov/bridge_test.go

```diff
@@ -1,48 +1,77 @@
 package gov_test
 
 import (
-	"github.com/dydxprotocol/v4-chain/protocol/lib"
 	"testing"
+	"time"
 
 	"github.com/cometbft/cometbft/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	govtypesv1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	"github.com/dydxprotocol/v4-chain/protocol/lib"
 	testapp "github.com/dydxprotocol/v4-chain/protocol/testutil/app"
 	"github.com/dydxprotocol/v4-chain/protocol/testutil/constants"
 	bridgetypes "github.com/dydxprotocol/v4-chain/protocol/x/bridge/types"
 	"github.com/stretchr/testify/require"
 )
 
-func TestUpdateEventParams(t *testing.T) {
-	genesisEventParams := bridgetypes.DefaultGenesis().EventParams
+var (
+	GENESIS_EVENT_PARAMS = bridgetypes.EventParams{
+		Denom:      "testdenom",
+		EthChainId: 123,
+		EthAddress: "0x0123",
+	}
+	GENESIS_PROPOSE_PARAMS = bridgetypes.ProposeParams{
+		MaxBridgesPerBlock:           10,
+		ProposeDelayDuration:         time.Minute,
+		SkipRatePpm:                  800_000,
+		SkipIfBlockDelayedByDuration: time.Minute,
+	}
+	GENESIS_SAFETY_PARAMS = bridgetypes.SafetyParams{
+		IsDisabled:  false,
+		DelayBlocks: 10,
+	}
+	INVALID_BRIDGE_AUTHORITY = constants.AliceAccAddress.String()
+)
 
+func TestUpdateEventParams(t *testing.T) {
 	tests := map[string]struct {
-		msg                      *bridgetypes.MsgUpdateEventParams
-		expectCheckTxFails       bool
-		expectSubmitProposalFail bool
-		expectedProposalStatus   govtypesv1.ProposalStatus
+		msg                       *bridgetypes.MsgUpdateEventParams
+		expectCheckTxFails        bool
+		expectSubmitProposalFails bool
+		expectedProposalStatus    govtypesv1.ProposalStatus
 	}{
 		"Success": {
 			msg: &bridgetypes.MsgUpdateEventParams{
 				Authority: lib.GovModuleAddress.String(),
 				Params: bridgetypes.EventParams{
-					Denom:      genesisEventParams.Denom + "updated",
-					EthChainId: genesisEventParams.EthChainId + 1,
-					EthAddress: genesisEventParams.EthAddress,
+					Denom:      "adv4tnt",
```

Comment:
For these updated fields that are used repeatedly, can you make a constant at the top of the file that describes these as modified event params so that the tests are easier to read?

For example, `ModifiedDenom`, `ModifiedEthAddress`, etc. 

This way it's easier to infer what you are doing in the code with the test cases. It would take the reader less time to understand the testing strategy that way,  because I don't have to manually check the values against the genesis params constant to see they are different.